src/reference/asciidoc/amqp.adoc

```diff
@@ -644,6 +644,17 @@ The `cacheMode` property (`CHANNEL` or `CONNECTION` is also included).
 .JVisualVM Example
 image::images/cacheStats.png[align="center"]
 
+[[auto-recovery]]
+===== RabbitMQ Automatic Connection/Topology recovery
+
+Since the first version of Spring AMQP, the framework has provided its own connection and channel recovery in the event of a broker failure.
+Also, as discussed in <<broker-configuration>>, the `RabbitAdmin` will re-declare any infrastructure beans (queues etc) when the connection is re-established.
+It therefore does not rely on the https://www.rabbitmq.com/api-guide.html#recovery[Auto Recovery] that is now provided by the `amqp-client` library.
+Spring AMQP now uses the `4.0.x` version of `amqp-client`, which has auto recovery enabled by default.
```

Comment:
The 4.0.0. client is compatible with earlier brokers; I don't think we need to "support" older clients in this new minor release (1.7.0).

We should also cherry-pick @acogoluegnes commit after this is merged bd4397a015d51bc5b7a5d954bf940396aa138eea